google-url-shortener.html

```diff
@@ -93,59 +121,58 @@
          * @type boolean
          * @default false
          */
-        auto: false,
-
-        readyForAction: function() {
-          apiLoaded_ = true;
-          this.fire('google-url-shortener-ready');
-        },
-
-        apiLoadError: function(event) {
-          this.fire('api-error', {
-            'error': {
-              'message': 'Error loading URL Shortener API',
-              'innerError': event.detail
-            }
-          });
-        },
-
-        longUrlChanged: function() {
-          if (this.auto) {
-            this.shorten();
+        auto: {
+          type: Boolean,
+          value: false
+        }
+      },
+
+      readyForAction: function() {
+        apiLoaded_ = true;
+        this.fire('google-url-shortener-ready');
+      },
+
+      apiLoadError: function(event) {
+        this.fire('api-error', {
+          'error': {
+            'message': 'Error loading URL Shortener API',
+            'innerError': event.detail
           }
-        },
+        });
+      },
 
-        /**
-         * Shortens the URL in `longUrl`.
-         *
-         * @method shorten
-         * @return {Object} Returns null.
-         */
-        shorten: function() {
-          if (apiLoaded_) {
-            if (this.longUrl) {
-              var request = this.$.urlshortener.api.url.insert(
-                  { 'resource': { 'longUrl': this.longUrl }});
-
-              request.execute(function(response) {
-                if (response && response.id) {
-                  this.fire('google-url-shorten', { shortUrl: response.id });
-                } else if (response && response.error) {
-                  this.fire('google-url-shorten-error',
-                      { 'error': response.error });
-                } else {
-                  this.fire('google-url-shorten-error',
-                      { 'error': 'Unknown error' });
-                }
-              }.bind(this));
-            }
-          }
-          return null;
+      longUrlChanged: function() {
+        if (this.auto) {
+          this.shorten();
         }
+      },
 
-      });
-    })();
+      /**
+       * Shortens the URL in `longUrl`.
+       *
+       * @method shorten
+        */
+      shorten: function() {
+        if (apiLoaded_) {
+          if (this.longUrl) {
+            var request = this.$.urlshortener.api.url.insert(
+                { 'resource': { 'longUrl': this.longUrl }});
+
+            request.execute(function(response) {
+              if (response && response.id) {
+                this.shortUrl = response.id;
+                this.error = '';
+                this.fire('google-url-shorten', { shortUrl: response.id });
```

Comment:
no spacing padding. When there's doubt, refer to https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml